package-lock.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Also I don't think we want to include the `package-lock.json` file here.

Reply:
Sure, we'll remove this file.